modules/appeals_api/app/swagger/appeals_api/v2/swagger.json

```diff
@@ -2,7 +2,7 @@
   "openapi": "3.0.0",
   "info": {
     "title": "Decision Reviews",
-    "version": "v2",
+    "version": "v0",
```

Comment:
This is our production swagger file. We do NOT want to be updating this with a new version 😨

Reply:
Oops 😓 sorry about that - I just updated this to leave `swagger.json` and `swagger_dev.json` totally alone for this PR. There are other updates to make to them because of changes in fixtures (as we talked about earlier), but I'll start on those in another PR where we can focus on them only.